client-flink/common/src/main/java/org/apache/celeborn/plugin/flink/utils/FlinkUtils.java

```diff
@@ -43,6 +43,15 @@ public class FlinkUtils {
           "remote-shuffle.job.compression.codec");
 
   public static CelebornConf toCelebornConf(Configuration configuration) {
+    if (Boolean.parseBoolean(
```

Comment:
@reswqa, the `CelebornTierFactory#setup` would also check this config.

Reply:
Yes, I will do the same stuff for hybrid shuffle if this PR merged.